From b6defdf22db38199f0de630cccc11c0c9b681350 Mon Sep 17 00:00:00 2001
From: Josef Schlehofer <pepe.schlehofer@gmail.com>
Date: Fri, 3 Mar 2023 01:44:16 +0100
Subject: [PATCH] luci-light: drop uhttpd dependency and its dependencies

In Turris OS, there is used as default web server Lighttpd, so it does
not make any sense to have additional web server from OpenWrt as there
would be two web servers and it would not be good to have it installed
side by side.

In the future, Turris OS developers should prepare luci-lighttpd package
as described in this issue:
https://gitlab.nic.cz/turris/os/build/-/issues/406

Signed-off-by: Josef Schlehofer <pepe.schlehofer@gmail.com>
---
 collections/luci-light/Makefile | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/collections/luci-light/Makefile b/collections/luci-light/Makefile
index d0d1997f04..78f9297de7 100644
--- a/collections/luci-light/Makefile
+++ b/collections/luci-light/Makefile
@@ -17,9 +17,7 @@ LUCI_DEPENDS:= \
 	+luci-mod-admin-full \
 	+luci-proto-ppp \
 	+luci-theme-bootstrap \
-	+rpcd-mod-rrdns \
-	+uhttpd \
-	+uhttpd-mod-ubus
+	+rpcd-mod-rrdns
 
 PKG_LICENSE:=Apache-2.0
 
-- 
2.34.1

